cartography/intel/gcp/dns.py

```diff
@@ -121,6 +114,134 @@ def get_dns_rrs(dns: Resource, dns_zones: List[Dict], project_id: str) -> List[R
             # raise
             return []
 
+@timeit
+def transform_rrs(rrsets: list, zone: dict, project_id: str):
+    list_rrs = []
+
+    for resource_record_set in rrsets:
+        resource_record_set['zone'] = zone['id']
+        resource_record_set['consolelink'] = gcp_console_link.get_console_link(
+            resource_name='dns_resource_record_set', project_id=project_id, dns_zone_name=zone['name'], dns_rrset_name=resource_record_set['name'])
+        resource_record_set[
+            "id"
+        ] = f"projects/{project_id}/resourceRecordSet/{resource_record_set.get('name',None)}"
+        list_rrs.append(resource_record_set)
+
+    return list_rrs
+
+@timeit
+def get_dns_keys(dns: Resource, zone: dict, project_id: str) -> List[Resource]:
```

Comment:
Please check it throughout the code. I see `dict` here again. It should be `Dict`.